dashboard/config/routes.rb

```diff
@@ -1017,6 +1017,11 @@
     get 'project_commits/get_token', to: 'project_commits#get_token'
     get 'project_commits/:channel_id', to: 'project_commits#project_commits'
 
+    # partial ports of legacy v3 APIs
+    get '/v3/channels/:channel_id/abuse', to: 'report_abuse#show_abuse'
+    post '/v3/channels/:channel_id/abuse/delete', to: 'report_abuse#reset_abuse'
+    patch '/v3/(:endpoint)/:encrypted_channel_id', constraints: {endpoint: /(animations|assets|sources|files|libraries)/}, to: 'report_abuse#update_file_abuse'
```

Comment:
can we get rid of the `patch` endpoint? in slack we talked about only get and reset being needed.

Reply:
I was wrong about that, I put this back in [3c4a8e4](https://github.com/code-dot-org/code-dot-org/pull/49960/commits/3c4a8e461ae1201caadf5b06957675cf2a444730)